src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -44,18 +46,20 @@ def get_snuba_query_string(snuba_query: SnubaQuery) -> str:
         SnubaQueryEventType.EventType.DEFAULT: "default",
         SnubaQueryEventType.EventType.TRANSACTION: "transaction",
     }
+    if not event_types:
+        event_types = snuba_query.event_types or []
 
-    if len(snuba_query.event_types) > 1:
+    if len(event_types) > 1:
         # e.g. (is:unresolved) AND (event.type:[error, default])
         event_types_list = [
-            SNUBA_QUERY_EVENT_TYPE_TO_STRING[event_type] for event_type in snuba_query.event_types
+            SNUBA_QUERY_EVENT_TYPE_TO_STRING[event_type] for event_type in event_types
         ]
         event_types_string = "(event.type:["
         event_types_string += ", ".join(event_types_list)
         event_types_string += "])"
     else:
         # e.g. (is:unresolved) AND (event.type:error)
-        snuba_query_event_type_string = SNUBA_QUERY_EVENT_TYPE_TO_STRING[snuba_query.event_types[0]]
+        snuba_query_event_type_string = SNUBA_QUERY_EVENT_TYPE_TO_STRING[event_types[0]]
```

Comment:
can we merge these cases? not sure the difference of the events_types_string if it's a list with 1 event_type vs the singular event type.

Reply:
The if and the else case? I left comments on what the strings look like. The only time it'll have more than one is if you pick both from the dropdown here: 
<img width="948" alt="Screenshot 2024-09-26 at 3 26 57 PM" src="https://github.com/user-attachments/assets/21c8757e-6cc8-467f-8228-7630b0f6673e">